### DISCUSSION TITLE
Local Embeddings

### DISCUSSION THREAD

**Discussion by NicolasDrapier** [mckaywrigley/chatbot-ui#1680] [Help]
Hi, just a quick question. I see that we can choose the embeddings provider, but I can't find if it is possible or not to provide my own embeddings.
If so, where am I supposed to go to add my local embeddings (use case with [text-embeddings-inference](https://github.com/huggingface/text-embeddings-inference))?

For example, how to set the URL of the server that is hosting the embedding model?

**Comment by tranhasimple** [comment]
I got the same problem :((
